### DISCUSSION TITLE
Select inside dialog

### DISCUSSION THREAD

**Discussion by renatoastra** [radix-ui/primitives#1814] [Help]
Hello guys, first of all sorry for my bad english, thats my frist post on github 🥺

Ok, the question is:

i have a dialog thats work perfect, now i put a select input inside it, i should use a new Portal to the  Select Primitive?
the dialog portal should work as a parent?

i notice when i click on the select field a new div appears inside the body pushing my content to left

**Comment by joaom00** [comment]
Hi @renatoastra,

You should use the `Portal` part exported from `Select`.
